src/config/ci-compaction-test/risingwave.toml

```diff
@@ -0,0 +1,51 @@
+[meta]
+enable_compaction_deterministic = true
+max_heartbeat_interval_secs = 600
+
+# The following is copied from ../risingwave.toml
```

Comment:
I'm worrying that it'll be hard to maintain in the future. 🥵 Can we introduce some mechanisms to merge/override/patch config files?

Reply:
oki

Reply:
Updated. PTAL